models.py

```diff
@@ -209,6 +209,10 @@ def do_write_corpus(self, update:bool):
             dir = os.path.dirname(os.path.join(self.ARGS.directory, ''))
             for file in os.listdir(dir):
                 filename = os.path.basename(file).split('.')[0]
+                if filename == "":
+                    eprint(f"ERROR: Corpus name is blank for file: {file}")
+                    return None
+                print("filename: ", filename)
```

Comment:
Please use eprint here too, unless this is a debugging message you are deleting.